staging/src/k8s.io/apiserver/plugin/pkg/audit/dynamic/dynamic_test.go

```diff
@@ -120,8 +136,26 @@ func TestDynamic(t *testing.T) {
 		err := checkForEvent(eventList1, testEvent)
 		require.NoError(t, err, "unable to find events sent to sink")
 	})
+	require.True(t, success) // propagate failure
```

Comment:
this is unexpected... was the test not failing before if one of these failed?

Reply:
yeah because they are sequentially dependent this will propagate the failure up to the parent